swift-sdk/IterableInboxCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  IterableInboxCell.swift
+//  swift-sdk
+//
+//  Created by Tapash Majumder on 4/11/19.
+//  Copyright © 2019 Iterable. All rights reserved.
+//
+
+import UIKit
+
+/// If you are creating your own Xib file you must
+/// connect the outlets.
+open class IterableInboxCell: UITableViewCell {
+    /// A "dot" view showing that the message is unread
+    @IBOutlet weak open var unreadCircleView: UIView?
+
+    /// The title label
+    @IBOutlet weak open var titleLbl: UILabel?
```

Comment:
What does the inbox UI layout look like without any title or subtitle or icon? Should we include a default title?

Reply:
If the title or subtitle is missing then it will be blank.
If the image is missing, the text will cover the image area as expected.
I think it would be difficult to come up with a default title that would make sense.